packages/rainbowkit/src/wallets/walletConnectors/walletConnectWallet/walletConnectWallet.ts

```diff
@@ -1,47 +1,31 @@
-import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
-import { getWalletConnectUri } from '../../../utils/getWalletConnectUri';
-import { isIOS } from '../../../utils/isMobile';
-import { Wallet } from '../../Wallet';
+import { RainbowKitWalletConnectParameters, Wallet } from '../../Wallet';
 import { getWalletConnectConnector } from '../../getWalletConnectConnector';
-import type { WalletConnectConnectorOptions } from '../../getWalletConnectConnector';
 
 export interface WalletConnectWalletOptions {
   projectId: string;
-  chains: Chain[];
-  options?: WalletConnectConnectorOptions;
+  options?: RainbowKitWalletConnectParameters;
 }
 
+// Used for maintance purposes for `connectorsForWallets` logic
+// wcId = "WalletConnect id"
+export const wcId = 'walletConnect';
```

Comment:
Probably unnecessary; we don't change this. Also creates a bit of a circular dependency